.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "pg_analytics/downloads"]
```

Comment:
Can we do this without a submodule please? The code should just be in `cargo-paradedb`. We plan to eventually move `cargo-paradedb` to a separate repository